File: SDKs/amazonaws.com/iotthingsgraph/2018-09-06/python/src/sdk/models/shared/updateflowtemplateresponse.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from . import flowtemplatesummary


@dataclass_json
@dataclass
class UpdateFlowTemplateResponse:
    summary: Optional[flowtemplatesummary.FlowTemplateSummary] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'summary' }})
    

File: SDKs/amazonaws.com/iotthingsgraph/2018-09-06/typescript/src/sdk/models/shared/updateflowtemplateresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { FlowTemplateSummary } from "./flowtemplatesummary";



export class UpdateFlowTemplateResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=summary" })
  summary?: FlowTemplateSummary;
}
